src/install/kafka/whatsNext.mdx

```diff
@@ -0,0 +1,4224 @@
+---
+componentType: default
+headingText: Metrics collected by the integration
+---
+
+## kafka-config.yml sample files [#instance-settings]
+
+<CollapserGroup>
+  <Collapser
+    id="zookeeper"
+    title="Zookeper discovery"
+  >
+    This configuration collects Metrics and Inventory including all topics discovering the brokers from two different JMX hosts :
```

Comment:
I notice in these that you have Metrics and Inventory capitalized, but I don't think we're supposed to. Also there seem to be spaces between the final words and the colon in some instances